CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ set(EG_DATA ${PROJECT_SOURCE_DIR}/tests/shape_eg_data CACHE STRING "")
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
I'm a bit nervous seeing C++17 being required in the main build path, whereas we only use that for testing. Could that be moved under a "if(BUILD_TESTING)" branch, so that people can possibly build core shapelib without a too recent C++ compiler?

Reply:
I guess I aready tried, but it did not work. It could work if the test code is a separate project. .. OK, will check,